packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2481,7 +2483,7 @@ export class ProjectService extends BaseService {
         const columns: VizColumn[] = [];
         let currentRowIndex = 0;
         let currentTransformedRow: ResultRow | undefined;
-        const valuesColumnReferences = new Set<string>(); // NOTE: This is used to pivot the data later with the same group by columns
+        const valuesColumnData = new Map<string, PivotValuesColumn>();
```

Comment:
ok cool. so we used to have uniq values of pivot column. but now the value is a key so it's still "uniq" 👍